package ar.edu.utn.frbb.tup.model.exception;

public class CorruptedDataInDbException extends InternalServerErrorException {

  public CorruptedDataInDbException(String message) {
    super("No pudo realizarse la operaci√≥n por un problema interno", 101);
    logger.error("Corrupted data in DB: " + message);
  }
}
